app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::UsersController < ApplicationController
+  def index
+    @users = SlackUser.order(:is_deleted).order(is_admin: :desc).order(:last_name, :email)
```

Comment:
You could also write this `order(:is_deleted, {is_admin: :desc}, :last_name, :email)`